v0/destinations/woopra/data/WoopraTrackConfig.json

```diff
@@ -0,0 +1,83 @@
+[
+   
+    {
+        "sourceKeys": "timestamp",
+        "destKey": "cv_originalTimestamp",
+        "sourceFromGenericMap":true
+    },
+    {
+        "sourceKeys": [
+            "traits.email",
+            "context.traits.email",
+            "properties.email"
+        ],
+        "destKey": "cv_email"
+    },
+    {
+        "sourceKeys": "userId",
+        "destKey": "cv_id",
+        "sourceFromGenericMap":true
+    },
+    {
+        "sourceKeys": "context.device.id",
+        "destKey": "cookie",
+        "sourceFromGenericMap":true
+    },
+    {
+        "sourceKeys": "timezone",
+        "destKey": "cv_timezone",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys": "context.ip",
+        "destKey": "cv_ip"
+    },
+    {
+        "sourceKeys": "sentAt",
+        "destKey": "timestamp",
+        "metadata": {
+            "type": "microSecondTimestamp"
+          }
+    },
+
+    {
+        "sourceKeys": "event",
+        "destKey": "event"
+      },
+      {
+        "sourceKeys": "properties.review_id",
+        "destKey": "ce_review_id"
+      },
+      {
+        "sourceKeys": "properties.product_id",
+        "destKey": "ce_product_id"
+      },
+      {
+        "sourceKeys": "properties.rating",
+        "destKey": "ce_rating"
+      },
+      {
+        "sourceKeys": "properties.review_body",
+        "destKey": "ce_review_body"
+      },
```

Comment:
same here. if woopra has not mentioned any of these. Lets not include it here. Rest of the properties look generic, they can remain. Lets filter out the generic properties and add ce prefix to all keys in properties and send out

Reply:
As I see from the documentation, we can send user parameters via track call as well. We need to add them in the payload as well.

Reply:
Lets also add the https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/#contextual-fields app, device, os and screen from here, as a session variable. Starting with `cs_< key name >`

Reply:
for app device and os we are already mapping directly i.e. cs_, ce_, cv_ is not required and for screen they have not mentioned anything and if we are passing its not showing anywhere.